content/06.methods.md

```diff
@@ -70,75 +69,71 @@ Tissues were lysed using a Qiagen TissueLyser II (Qiagen) with 2×30 sec at 18Hz
 Both tissue and cell pellets processes included a CHCl3 extraction and were run on the QIACube automated platform (Qiagen) using the AllPrep DNA/RNA/miRNA Universal kit (cat# 80224, Qiagen).
 Blood was thawed and treated with RNase A (cat#, 19101, Qiagen); 0.4-1ml was processed using the Qiagen QIAsymphony automated platform (Qiagen) using the QIAsymphony DSP DNA Midi Kit (cat# 937255, Qiagen).
 DNA and RNA quantity and quality was assessed by PerkinElmer DropletQuant UV-VIS spectrophotometer (PerkinElmer) and an Agilent 4200 TapeStation (Agilent, USA) for RIN and DIN (RNA Integrity Number and DNA Integrity Number, respectively).
```

Comment:
this may also be tough as we did not perform extractions in house (our biorespository core did this and they are not currently authors).... but should they be? this probably goes back decades....

Reply:
leaving passive